samples/stock/demo/all-indicators/demo.js

```diff
@@ -0,0 +1,147 @@
+            top: '75%',
+            height: '25%'
+        }],
+        series: [{
+            type: 'candlestick',
+            id: 'aapl',
+            name: 'AAPL',
+            data: data
+        }, {
+            type: 'column',
+            id: 'volume',
+            name: 'Volume',
+            data: volume,
+            yAxis: 1
+        }]
+    }, function () {
+        document.getElementById("overlays").addEventListener("change", function (e) {
+            const seriesToAdd = e.target.value;
+
+            // Destroy previously existing series.
+            if (chart.series[2] && chart.series[2].yAxis.options.index === 0) {
+                chart.series[2].destroy(false);
+            }
+            if (chart.series[3] && chart.series[3].yAxis.options.index === 0) {
+                chart.series[3].destroy(false);
+            }
+
+            if (seriesToAdd !== 'none') {
+                chart.addSeries({
+                    type: seriesToAdd,
+                    linkedTo: 'aapl',
+                    showInLegend: true
+                }, false);
+            }
+            chart.redraw();
+        });
+
+        document.getElementById("oscilators").addEventListener("change", function (e) {
+            const seriesToAdd = e.target.value;
+
+            // Add third axis if needed.
+            if (chart.yAxis.length === 3) {
+                chart.yAxis[0].update({
+                    height: '55%'
+                }, false);
+                chart.yAxis[1].update({
+                    top: '60%',
+                    height: '15%'
+                }, false);
+                chart.addAxis({
+                    top: '80%',
+                    height: '20%',
+                    opposite: true,
+                    title: {
+                        text: null
+                    }
+                }, false);
+            }
+
+            // Destroy previously existing series.
+            if (chart.series[2] && chart.series[2].yAxis.options.index === 3) {
+                chart.series[2].destroy(false);
+            }
+            if (chart.series[3] && chart.series[3].yAxis.options.index === 3) {
+                chart.series[3].destroy(false);
+            }
+
+            // Remove axis if not necessary.
+            if (seriesToAdd === 'none') {
+                chart.yAxis[3].remove(false);
+
+                chart.yAxis[0].update({
+                    height: '70%'
+                }, false);
+                chart.yAxis[1].update({
+                    top: '75%',
+                    height: '25%'
+                }, false);
+            } else {
+                if (seriesToAdd === 'aroon') {
+                    chart.addSeries({
+                        yAxis: 3,
+                        type: 'aroon',
+                        linkedTo: 'aapl',
+                        color: 'green',
+                        lineWidth: 1,
+                        showInLegend: true,
+                        aroonDown: {
```

Comment:
Can we simplify it? I mean, let's treat `aroon` as any other tech. indicator, without defining styles (just use defaults)